tests/Doctrine/Tests/Common/Inflector/InflectorTest.php

```diff
@@ -305,5 +305,97 @@ public function testUcwordsWithCustomDelimeters()
     {
         $this->assertSame('Top-O-The-Morning To All_Of_You!', Inflector::ucwords( 'top-o-the-morning to all_of_you!', '-_ '));
     }
+
+    /**
+     * @param $expected
+     * @param $word
+     *
+     * @dataProvider dataStringsTableize
+     * @return void
+     */
+    public function testTableize($expected, $word)
+    {
+        $this->assertSame($expected, Inflector::tableize($word));
+    }
+
+    /**
+     * Strings which are used for testTableize.
+     *
+     * @return array
+     */
+    public function dataStringsTableize()
+    {
+        Inflector::reset();
```

Comment:
This does not make sense in the data provider. All data providers are called at the beginning of the PHPUnit run, not just before the test